src/shogun/converter/Isomap.h

```diff
@@ -39,6 +49,9 @@ class CDistance;
  *
  * To use this converter with static interfaces please refer it by
  * sg('create_converter','isomap',k);
+
+ * cf. http://tapkee.lisitsyn.me/
+ * cf. https://en.wikipedia.org/wiki/Isomap
```

Comment:
Mm I just saw there is already a reference for another paper at the beginning of the class description. I think the one in the link above is more suitable since it is an older paper but I am fine leaving it as it is as well.